packages/api/pages/api/languages/import/options.ts

```diff
@@ -0,0 +1,33 @@
+import { GetLanguageImportOptionsResponseBody } from '@translation/api-types';
+import createRoute from '../../../../shared/Route';
+
+const IMPORT_SERVER = 'https://hebrewgreekbible.online';
+
+export default createRoute()
+  .get<void, GetLanguageImportOptionsResponseBody>({
+    async handler(req, res) {
+      try {
+        // Request the server root page.
+        const response = await fetch(IMPORT_SERVER);
+        const html = await response.text();
+        // Parse the HTML for the language options.
+        const regex = /var glossLanguageNames\s*=\s*\[([\s\S]*?)\];/;
+        const matches = html.match(regex);
+        if (matches && matches[1]) {
+          const languageNames = matches[1]
+            .split(',')
+            .map((name: string) => name.trim().replace(/['"]+/g, ''))
+            .filter((name: string) => name.length > 0);
+          // Return the options.
+          res.ok({ data: languageNames });
+          return;
+        }
+      } catch (error) {
+        console.error('Error retrieving options:', error);
+        // TODO: it would probably be better to throw an actual error?
```

Comment:
How should this be handled? Is sending an empty list of options acceptable?

Reply:
If there is an error loading the data, we should definitely return one, but I think it should be a  500 error since it isn't an issue with the client's request, but on the "server" side. I think for now, we can leave it unhandled by removing the try catch, by default it should return 500. It's something only we would be able to fix, so if we ever see it on this route, we'll know how to track it down.